### ISSUE TITLE
Run /LightGBM-master/examples/python-guide/sklearn_example.py  get some UserWarnings

### ISSUE THREAD

**Issue by datahonor** [Microsoft/LightGBM#341]
Hello, when run the **python /LightGBM-master/examples/python-guide/sklearn_example.py** file, the result of the running have same UserWarning: Using Pandas (default) integer column names, not column indexes. You can use indexes with DataFrame.values.
I try to modify the sklearn_example.py, as follows:
#######################################
df_train = pd.read_csv('../regression/regression.train', header=None,names=["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z","aa"], sep='\t')
df_test = pd.read_csv('../regression/regression.test', header=None,names=["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z","aa"], sep='\t')

y_train = df_train["a"]
y_test = df_test["a"]
X_train = df_train.drop("a", axis=1)
X_test = df_test.drop("a", axis=1)
#######################################
The UserWarnings disappear, but the result of the running is different from previous results.

## Environment info
Operating System: Ubuntu 14
CPU: 
C++/Python/R version: Python

**Comment by wxchan** [comment]
can you paste the different results here?

**Comment by wxchan** [comment]
I planned to let you see the warning in example, but it appears indeed a little annoying. I will remove it later.

**Comment by dahuan1** [comment]
Sorry, I know the reason. i should modify:
**names=["a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z","aa","ab","ac"]**
Thanks.
